README.md

```diff
@@ -1 +1,24 @@
 # DigitalBank
+CASOS PRUEBA TOMADOS EN CUENTA:
+
+Log-in:
+
+Utilizando las credenciales validas aportadas en la descripción del proyecto, decidí necesario realizar los siguientes casos:
```

Comment:
Also, remember that we defined the convention of **everything in english**, this includes everything on the repo